### ISSUE TITLE
support many sizes of screen not just 640 x 480

### ISSUE THREAD

**Issue by dougmencken** [dougmencken/HeadOverHeels#30]
> The deauflt 640x480 window is too small, the minimum should be 800x600, even better 1024x769. In 16/9 monitors in fullsrceen everything is flat, since the original game is adapted to 3/4 monitors (TV). It would be useful to make the output screen adjustable.

https://github.com/dougmencken/HeadOverHeels/issues/25#issuecomment-379858665

**Comment by dougmencken** [comment]
Yeah, that’s nice idea

![second](https://user-images.githubusercontent.com/3457110/38542315-17431df8-3c91-11e8-986d-dffbc0473a0c.png)

**Comment by dougmencken** [comment]
it’s pretty playable at 800 x 600 yet

![game at 800 x 600](https://user-images.githubusercontent.com/3457110/38572639-263780cc-3ce3-11e8-9594-e570d349045d.png)

**Comment by dougmencken** [comment]
There’s no option in menu for this yet, you need to pass `--width=foo` and `--height=bar` at the moment of game’s invocation

**Comment by attuska** [comment]
OK.

**Comment by dougmencken** [comment]
The problem is which ones to present to user in menu. “ Hard coded ” lists like 640 x 480, 800 x 600, 1024 x 600, 1280 x 720? That wouldn’t satisfy everyone

**Comment by attuska** [comment]
The basic graphics size remains in window mode as in 640x480 mode, which is independent of the size of the window.

**Comment by dougmencken** [comment]
Yes, I don’t stretch ( scale ) sprites of game, and I doubt I would

And you may enjoy huge rooms without scrolling and dealing with camera

![sala triple](https://user-images.githubusercontent.com/3457110/38574286-ff09fa20-3ce7-11e8-9fba-3bd0e2c80c78.png)

**Comment by dougmencken** [comment]
I added icon 128 x 128 to repository

Plus here is 512 x 512

![head_huge](https://user-images.githubusercontent.com/3457110/38607722-f3a32324-3d68-11e8-9188-5067218ed283.png)

and vector

[head_huge.svg.zip](https://github.com/dougmencken/HeadOverHeels/files/1898135/head_huge.svg.zip)

**Comment by dougmencken** [comment]
with https://github.com/dougmencken/HeadOverHeels/commit/58c250603c1be33868608db51451b8e2cb240909
the only thing that remains to do is the option in the video menu

**Comment by dougmencken** [comment]
to do

✓ adapt existing slides to the new window size (without relaunching the game)
✓ the option to choose custom (any) size

**Comment by dougmencken** [comment]
✓ ~adapt existing slides to the new window size~

**Comment by dougmencken** [comment]
✓ ~the option to choose custom (any) size~

**Comment by dougmencken** [comment]
to do

✓ better background for slides in the present graphics set

**Comment by dougmencken** [comment]
with e4445573523c808d6a2ad15e31d2738faefec977, the background is okay for 1920×1080

**Comment by dougmencken** [comment]
to do

✓ make the default screen size to be bigger than 640×480

**Comment by dougmencken** [comment]
oooops
![after setting custom size](https://github.com/user-attachments/assets/cbf35fd6-87d6-4dda-9bb6-1d98007bebc9)
